getting-started/Installation/system-requirements.md

```diff
@@ -45,12 +45,14 @@ Supported Development Tools and Environments:
         
 * __.NET Framework 4.0, or later__ - download .NET 4.0 from [here](http://www.microsoft.com/downloads/details.aspx?FamilyID=9cfb2d51-5ff4-4491-b0e5-b386f32c0992&displaylang=en), .NET 4.5 from [here](http://www.microsoft.com/en-us/download/details.aspx?id=30653), .NET 4.5.1 from [here](http://www.microsoft.com/en-us/download/details.aspx?id=40779), .NET 4.5.2 from [here](http://www.microsoft.com/en-us/download/details.aspx?id=42642) or .NET 4.6 from [here](https://www.microsoft.com/en-us/download/details.aspx?id=48130).
 
-* __.NET Core 3.0__ - download .NET 3.0 from [here](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+* __.NET Core 3.0, or later__ - download .NET Core 3.0 from [here](https://dotnet.microsoft.com/download/dotnet-core/3.0) or .NET Core 3.1 from [here](https://dotnet.microsoft.com/download/dotnet-core/3.1).
```

Comment:
It is not necessary to add all the versions, 'later' is enough. Limit the link to https://dotnet.microsoft.com/download/dotnet-core so the users can select the version they need.